docs/benchmarks/fb-oss-performance.md

```diff
@@ -1,5 +1,7 @@
 # Facebook Workload
 
+> **Warning:** This benchmark is deprecated in CloudSuite 4 and not maintained anymore. However, you can still download the binary image with the command in this document and run the experiment.
```

Comment:
not maintained --> will not be maintained or we will not maintain.
command --> commands